project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_users/c4m_og_invite_users.module

```diff
@@ -108,3 +108,92 @@ function c4m_og_invite_users_mail_tokens(array &$replacements, array $data, $opt
   $replacements['[group:accept_invite_url]'] = $answer_link;
   $replacements['[site:name]'] = variable_get('site_name', 'Capacity 4 More');
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add a subject field to the invitation form.
+ */
+function c4m_og_invite_users_form__og_invite_form_alter(&$form, &$form_state, $form_id) {
+  $form['subject'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Subject'),
+    '#weight' => 10,
+  );
+
+  $form['message']['#weight'] = 20;
+  $form['#submit'][0] = 'c4m_og_invite_users_form_submit';
```

Comment:
Yes, I want to replace the submit handler function to alter some functionality (Add the subject input to be processed).